src/nat/agent/tool_calling_agent/register.py

```
@@ -115,9 +115,7 @@ async def _response_fn(input_message: ChatRequest) -> str:
             return str(output_message.content)
         except Exception as ex:
             logger.exception("%s Tool Calling Agent failed with exception: %s", AGENT_LOG_PREFIX, ex)
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)